turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -427,7 +427,10 @@ struct AnalysisState<'a> {
     origin: ResolvedVc<Box<dyn ResolveOrigin>>,
     compile_time_info: ResolvedVc<CompileTimeInfo>,
     var_graph: &'a VarGraph,
-    define_process_cwd: bool,
+    /// Whether to allow tracing to reference files from the project root. This is used to prevent
+    /// random node_modules packages from tracing the entire project due to some dynamic
```

Comment:
in a shipped node module if i wanted to read a package relative file and get traced i would need to do something like `readFileSync(join(__dirname, 'my-file'))` and that _would_ work, right?